.travis.yml

```diff
@@ -39,6 +40,9 @@ jobs:
     ###########################################
 
     # Disable Async Hooks
```

Comment:
Test, please ignore me:

```suggestion
    # Disable Async-Hooks
```

Reply:
Made this review to help github staff fix the a bug in the suggested change feature - just ignore